demo/psa/prov-verif-e2e/README.md

```diff
@@ -0,0 +1,152 @@
+# PSA Demonstration Steps
+
+This document explains all the steps one need to follow to successfully run
+end to end PSA Demonstration
+
+## Preconditions
+
+* One need to install `jq` and `curl`.
+
+## Provisioning pipeline setup
+
+* Install `cocli tool` using following command
+
+In a new bourne shell session
```

Comment:
Technically, the following snippet does not need to be run a bourne shell -- since you're only invoking go, and not using any shell features, any shell would do.

Rather than specifying this here (or every time it's appropriate), I would just add a note to the top of the file. Something along the lines of 

```markdown
> **Note**
> Commands below assume you're using a Bourne-compatible shell. 
> Please adjust them appropriately if that is not the case.
```